src/main/java/com/coveo/pushapiclient/PlatformClient.java

```diff
@@ -567,7 +570,22 @@ private String[] getAuthorizationHeader() {
   }
 
   private String[] getContentTypeApplicationJSONHeader() {
-    return new String[] {"Content-Type", "application/json", "Accept", "application/json"};
+    MavenXpp3Reader reader = new MavenXpp3Reader();
+    String version = "";
+    try {
+      Model model = reader.read(new FileReader("pom.xml"));
+      version = model.getVersion();
+    } catch (Exception e) {
```

Comment:
Try decide whether we should try to gracefully fail or make a slightly larger change such that `getContentTypeApplicationJSONHeader` throws an error, and other methods need to try-catch it. Thinking of going with the former so that `version` just falls back to `N/A` or something 🤷‍♂️